prime-router/docs/vpn.md

```diff
@@ -41,6 +41,38 @@ $ kill -${SIGNAL} $(cat "/tmp/${USER}/openvpn.pid")
 # On termination, openvpn will remove the PID file
 ```
 
+## Windows
+### VPN DNS Resolution
+There is an issue where the DNS server for the VPN adapter is not used when resolving hostnames.  This is due to the interface metric of the VPN loosing to the metric of your normal network adapter.  Run the following command to look up the IP of a server in the Azure environment and test if this is an issue .  For example:
+`nslookup pdhstaging-pgsql.postgres.database.azure.com`
```

Comment:
Is there a better host I can use than this?

Reply:
They're going to be dependent on which environment you connect to; for prod, this would be `pdhprod-pgsql.postgres.database.azure.com`

Reply:
Maybe?

```suggestion
`nslookup pdh<env>-pgsql.postgres.database.azure.com`
```